semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -249,6 +249,13 @@ object Main {
         //   val spre = loop(pre)
         //   val smix = loop(gmix)
         //   Some(s.Type(tag = stag, superType = Some(s.SuperType(spre, smix))))
+        case ConstantType(underlying: Type) =>
+          loop(underlying).map { sarg =>
+            val stag = t.TYPE_REF
+            val ssym = "_root_.java.lang.Class#"
+            val sargs = sarg :: Nil
+            s.Type(tag = stag, typeRef = Some(s.TypeRef(None, ssym, sargs)))
+          }
```

Comment:
The prefix must be non-empty here as per the updated spec, but I'll take care of that myself when working on implementing the spec.